go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/opennota/wd v0.0.0-20180911144301-b446539ab1e7 // indirect
 	github.com/russross/blackfriday v2.0.0+incompatible // indirect
 	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
-	google.golang.org/genproto v0.0.0-20200207204624-4f3edf09f4f6
+	google.golang.org/genproto v0.0.0-20200211111953-2dc5924e3898
```

Comment:
Can you revert the changes to the `go.mod` and `go.sum`, please? We have a bot that updates the deps.